server/services/draftsService.ts

```diff
@@ -32,7 +33,11 @@ export default class DraftsService {
     const dto = req.session[this.sessionKey(id)]
 
     if (dto === undefined) {
-      throw new Error(`Draft with ID ${id} not found in session`)
+      throw createError(404, new Error(`Draft with ID ${id} not found in session`), {
```

Comment:
minor nit: you can just pass the string as the second arg

Reply:
do we wanna be alerted to users hitting this error? if so, we should return 500 here, just because 404s are ignored in our monitoring tools

Reply:
Yeah, I think I'd like to see how often it happens for now, to see how much we need to invest in better error messaging. So let's make it a 500 to start